alexandria/core/management/commands/fix_copied_file_references.py

```diff
@@ -0,0 +1,90 @@
+import botocore
+from django.core.management.base import BaseCommand
+from tqdm import tqdm
+
+from alexandria.core.models import File
+
+
+class Command(BaseCommand):
+    help = "Fixes copied file references by creating a new copy."
+
+    def add_arguments(self, parser):
+        parser.add_argument("--dry", dest="dry", action="store_true", default=False)
+
+    def handle(self, *args, **options):
+        filename_counts = {}
+        for tmp_file in tqdm(
+            File.objects.filter(variant="original").order_by("created_at")
+        ):
+            filename = tmp_file.content.name
+            if filename in filename_counts:
+                filename_counts[filename].append(tmp_file.pk)
+            else:
+                filename_counts[filename] = [tmp_file.pk]
```

Comment:
If you use a `defaultdict` for this, you can get rid of the `if` check here, and just call `.append()` directly.
For example:
```python
from collections import defaultdict
foo = defaultdict(list)
foo["test"].append(1)
...
```

Reply:
Changed to defaultdict